x/lend/keeper/asset.go

```diff
@@ -135,3 +135,15 @@ func (k *Keeper) GetAssetForDenom(ctx sdk.Context, denom string) (asset types.As
 
 	return k.GetAsset(ctx, id.GetValue())
 }
+
+func (k *Keeper) GetBalancesForModule(ctx sdk.Context, module string) (balances string) {
+	if module != "osmo" && module != "cmdx" && module != "atom" {
```

Comment:
move string to constants

Reply:
@karthik340 use types.ModuleAcc1 and other module accounts previously defined in the module.

Reply:
@karthik340 also pls update module name to types.ModuleAcc_{DenomType} and include it in this pr.

Reply:
is there a way to handle it, if new asset is added via proposal ?

Reply:
Currently, we have to do a software upgrade proposal for adding a new module account.

Reply:
There should be a way, like including it in param and use that. This can be done via params update instead of software upgrade.

Reply:
okay I get it, but it has to be included in app in maccpersmissions.